bramkouwen/Neatly-In-Motion
# Creating Base of How-To Page

## What does this PR do?
* It creates how_to.html 
* It creates the basic page layout for how_to.html

## How do you check this PR?
* Pull down the branch
* Check the added how_to.html file  
* Check the base page layout

## Screenshots 
None.
